app/src/main/java/com/sma2/sma2/FeatureExtraction/Tapping/Tapping_feature_Activity.java

```diff
@@ -99,12 +114,11 @@ protected void onCreate(Bundle savedInstanceState) {
         double delay_hits_two= delay_hits_Tapping_one(Count_Touch_one,Delay_one);
         float Hist_Porcentage=Count_ladybug_one(Count_Touch_one);
         float dimention=pixeltomm((float)average_funtion(Distance_one));
-        float perc_distance= (float) distancetoperc(dimention);
-
+        float perc_distance=100/(1+dimention);
```

Comment:
why did you change this???

Such conversion between the distance and the percentage is outdated, 
please review distancetoperc() function